queries.sql

```diff
@@ -5,4 +5,82 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = '1';
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+begin
+
+UPDATE animals
+SET species = 'unspecified'
+ROLLBACK
+
+begin
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species is null;
+
+commit
+
+SELECT * from animals
```

Comment:
- [ ] Good job adding the select statement after the `commit` statement to verify the change. but as mentioned earlier we need a command to verify if the changes in the transaction were made **before** and after commit. for that alongside the `SELECT * from animals` after `commit` please add one before the `commit` statement.